content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -0,0 +1,231 @@
+=================
+PDF quote builder
+=================
+
+The *PDF Quote Builder* in Odoo *Sales* provides the opportunity to export a fully-custom PDF file
+for quotes, showcasing various information and design elements, instead of just the price and total.
+
+The PDF Quote Builder can be used to add header pages, product descriptions, and footer pages to a
+quote.
+
+Having a customized PDF in quotes provides a heightened conclusion to the shopping experience for
+customers, and adds an elegant level of professionalism to a company.
```

Comment:
I think this quick sentence adds a bit of context to any curious reader who may have just wandered into this doc, wondering what the benefits of a custom PDF in a quote could be...but, if you think it would be best to remove it, just let me know @meng-odoo and I'll get rid of it right away.